components/icon-buttons.jsx

```diff
@@ -0,0 +1,21 @@
+var React = require('react');
+
+var IconButtons = React.createClass({
+  render: function() {
+    var links = Array.isArray(this.props.children) ? this.props.children : [this.props.children];
```

Comment:
Rather than creating this and mapping over it, maybe use [`React.Children.map()`](http://facebook.github.io/react/docs/top-level-api.html#react.children.map)?